content/blog/2021/04/2021-04-20-tekton-plugin.adoc

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: "Easily reuse Tekton and Jenkins X from Jenkins"
+tags:
+- jenkins-x
+- kubernetes
+- pipeline
+- tekton
+- gitops
+authors:
+- jstrachan
+description: >
+  This post gives an overview of the new Tekton plugin for Jenkins and how it can help you reuse the power of Tekton pipelines, the Tekton Catalog and the automated CI/CD from Jenkins X
+---
+
+== What is Tekton?
+
+image:/images/tekton-plugin/logo.png[TektonLogo,width=200,role="right"]
+
+link:https://tekton.dev/[Tekton] is a powerful and flexible open-source framework for creating CI/CD systems, allowing developers to build, test, and deploy across cloud providers and on-premise systems.
+
+=== Why use Tekton?
+
+Tekton pipelines have a number of benefits:
+
+* they are cloud native and designed from the ground up for kubernetes
+* each Tekton `Pipeline` is fully declarative and completely self described; it does not depend on any separate out of band Jenkins controllers, plugins or plugin/controller configurations
+* each `Pipeline Task` runs as a stand alone kubernetes link:https://kubernetes.io/docs/concepts/workloads/pods/[Pod] which is completely independent of any other pods and pipelines and are fully scheduled by Kubernetes to maximise resilience and optimize resource usage. A bad pipeline cannot take down another one & the kubernetes scheduler manages them all
+* each step can be any command in any container image with whatever secrets, volume mounts, environment variables and resource limits you need
+* there is no need to bundle a JVM or Jenkins Remoting container into the pod so you can keep resources and cost down
+
+== Why use Jenkins and Tekton together?
+
+link:https://www.jenkins.io/[Jenkins] is the most popular open source automation server around. Lots of developers use it every day to get things done.
+
+Jenkins can now be used to automate link:https://tekton.dev/[Tekton] pipelines too which helps teams digitally transform to more cloud native solutions for their CI and CD.
+
+
+== Introducing the Tekton Plugin for Jenkins
+
+The link:https://github.com/jenkinsci/tekton-client-plugin[tekton plugin for jenkins] lets you easily use Jenkins to automate creating and running Tekton pipelines.
+
+Its a single Jenkins plugin to install - so it's easy to use.
```

Comment:
Would it makes sense to reference https://cd.foundation/blog/2020/11/05/bridging-the-gap-with-tekton-client-plugin-for-jenkins/ from @waveywaves or to embed the video?

Reply:
I've added a link to the blog post and to @waveywaves